### PR TITLE
[Edit] Edits to proof-by-strong-induction.md

### PR DESCRIPTION
**Pull request by noahpgordon**
### Description

Edited file proof-by-strong-induction.md

-Corrected basic misunderstanding of the relationship between strong and weak induction (both can have multiple base cases, contra previous entry).
-Clarified example proof.
-Added explanation of the logic of strong induction.

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [X] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Review by SSwiniarski — APPROVED** [review]
@noahpgordon Looks good to me. I'm sending for a second review.

**Comment by noahpgordon** [comment]
@yangc95 I made some changes to the recent edits. The most significant were that I changed the description so that it reflects the difference between strong induction and regular induction, and I eliminated the reference to recursion, as it was somewhat inaccurate. (Recursion is not used to prove the inductive step in a strong induction, although there are clear similarities between the logic of induction as a whole and the technique of recursion.)

File: content/discrete-math/concepts/proofs/terms/proof-by-strong-induction/proof-by-strong-induction.md [hunk 1]
```diff
@@ -14,43 +14,59 @@ CatalogContent:
...
+
+In a weak mathematical induction, the inductive step involves showing that if some element _n_ has a property, then the successor element _n + 1_ must also have that property. In a strong mathematical induction, the inductive step involves showing that if all elements up to and including _n_ have some property, then _n + 1_ has that property as well.
 
 ## Steps for Strong Induction
 
-1. _Base Cases_: Identify multiple smallest possible cases for which the statement is true.
-2. _Induction Hypothesis_: Suppose that the statement is true for some integer `k`, then the statement must be true for `k+1`.
-3. _Proof_: Prove that the induction hypothesis is true for `k+1`.
+1. Base Step: Prove that the first element or elements in the series have some property.
+2. Inductive Step: Prove that if _n_ and all elements prior to n have the relevant property, then _n + 1_ has that property.
```
Comment (dakshdeepHERE) [inline content/discrete-math/concepts/proofs/terms/proof-by-strong-induction/proof-by-strong-induction.md:24]:
```suggestion
1. `Base Step`: Prove that the first element or elements in the series have some property.
2. `Inductive Step`: Prove that if _n_ and all elements before n have the relevant property, then _n + 1_ has that property.
```

**Review by dakshdeepHERE — CHANGES_REQUESTED** [review]

**Review by dakshdeepHERE — APPROVED** [review]
This looks good for merging 🚀

**Pull request merged**